dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -92,7 +92,7 @@
         <AD id="decodeUrlPlusSign"
             name="%decodeUrlPlusSign.name" 
             description="%decodeUrlPlusSign.desc" 
-            required="false" type="Boolean" default="false" />
+            required="false" type="Boolean" default="true" />
```

Comment:
I don't think you should change the default value.  It should have been "false" (i.e not decode plus into space character in the URI).

With this fix, we may break some customers but I also don't know who can operate on a "space" in their URIs.  Even so, they can use the property the change the default behavior.